contents/docs/surveys/viewing-results.mdx

```diff
@@ -48,11 +48,17 @@ There are 3 survey-related events you create insights from:
 
 When creating an insight, select any of these events. To see data for a specific survey, create a filter using the `Survey ID` or `Survey Name` properties.
 
-Depending on your [question type](/docs/surveys/creating-surveys#question-type), you may also find it helpful to breakdown results by `Survey Response` (e.g., for number rating or choice selection).
+Depending on your [question type](/docs/surveys/creating-surveys#question-type), you may also find it helpful to breakdown results by the response.
+
+**You can do that by usign the built-in SQL function `getSurveyResponse(question_index, question_id, is_multiple_choice)`**.
```

Comment:
I think we should mention the breaking change and the usage of this function in the docs/surveys/troubleshooting as well, its a common problem and a common place to look for solutions